.travis/push-docker-image

```diff
@@ -0,0 +1,18 @@
+#! /usr/bin/env bash
+
+set -e
```

Comment:
-v maybe so that we can see the output?

Reply:
ah or do you just rely on the echo commands?

Reply:
I've got a `set -x` in that script later, which will show the commands being executed.
`set -e` will make the script exit on errors. Run `help set` for more info.